Formula/nvm.rb

```diff
@@ -39,6 +39,6 @@ def caveats; <<-EOS.undent
     output = pipe_output("NODE_VERSION=homebrewtest #{prefix}/nvm-exec 2>&1")
     assert_no_match /No such file or directory/, output
     assert_no_match /nvm: command not found/, output
-    assert_match "N/A: version \"homebrewtest\" is not yet installed", output
+    assert_match /N\/A: version "N\/A" is not yet installed/, output
```

Comment:
I see. I found other examples of assert_match with a regex so I just assumed.

https://github.com/Homebrew/homebrew-core/search?utf8=%E2%9C%93&q=assert_match